src/_gettsim/taxes/zu_verst_eink/freibetraege.py

```diff
@@ -335,27 +335,50 @@ def sonderausgaben_betreuung_y_tu(
     return float(out)
 
 
-def eink_st_kinderfreib_y_tu(
-    anz_kinder_mit_kindergeld_tu: float,
-    anz_erwachsene_tu: int,
+def eink_st_kinderfreib_anspruch(
+    kindergeld_anspruch: bool,
+):
+    """Check if child is eligible for child allowance.
+
+    Parameters
+    ----------
+    kindergeld_anspruch
+        See :func:`kindergeld_anspruch`.
+
+    Returns
+    -------
+
+    """
+    return kindergeld_anspruch
+
+
+def eink_st_kinderfreib_kind_y(
```

Comment:
While I like the general idea of what you are suggesting, here it is confusing. There simply is no notion of Kinderfreibetrag at the child level, it is always linked to the taxpayers.

I suggest we aggregate `kindergeld_anspruch` at the parent level and only then add up the payment values here.

Reply:
I agree, but keeping something like

```python
def eink_st_kinderfreib_anspruch_kind(
    kindergeld_anspruch: bool,
) -> bool:
    return kindergeld_anspruch
```

and summarizing this internally based on `p_id_elternteil_x` to `eink_st_kinderfreib_anspruch_eltern: int` makes it easier to use the same infrastructure for both transfers and Freibeträge.

Reply:
See my previous point. We can do all that with `kindergeld_anspruch`, really no need to create a synonym.

Reply:
My point is that for transfers we would aggregate `x_kind` to `x_eltern` but with your suggested change, Freibeträge would be special because we aggregate `y_kind` to `x_eltern`. I wanted to build on the logic of the aggregation suffixes that we already have. But of course, we can handle this special case internally.

Reply:
True, that's a good approach.